### PR TITLE
[CI] Cleanup stale logs for auto-tuning

### PR DESCRIPTION
**Pull request by tqchen**
cc @merrymercy @areusch @junrushao1994

### CODE REVIEW

**Comment by tqchen** [comment]
should fix the recent ci flaky issues

**Review by junrushao1994 — APPROVED** [review]

File: tests/scripts/task_python_docs.sh [hunk 1]
```diff
@@ -40,7 +40,9 @@ rm -rf docs/doxygen
 
 # prepare auto scheduler tutorials
 rm -rf tutorials/auto_scheduler/*.json
+rm -rf tutorials/get_started/*.json
 cp -f tutorials/auto_scheduler/ci_logs/*.json tutorials/auto_scheduler
+cp -f tutorials/auto_scheduler/ci_logs/*.json tutorials/get_started
```
Comment (areusch) [inline tests/scripts/task_python_docs.sh:45]: why this line?

**Review by areusch — CHANGES_REQUESTED** [review]

File: tests/scripts/task_python_docs.sh (hunk 1 above)
Comment (tqchen) [reply tests/scripts/task_python_docs.sh:45]: the particular flaky error was tune matmul in get started, the tutorial get moved over there, so we need some log to help prevent flaky issue

Comment (areusch) [reply tests/scripts/task_python_docs.sh:45]: in tune_matmul, it produces its own log file, no? that's why the target with new default option caused troubles in CI.

Comment (tqchen) [reply tests/scripts/task_python_docs.sh:45]: it is for the case where you get too few rounds but no valid record i believe

**Review by areusch — APPROVED** [review]
Ok @tqchen, I don't want to hold up a CI fix, but would be great to document this in a follow on either in the test script or tutorial.

**Pull request merged**